docs/sources/flow/reference/components/prometheus.integration.redis.md

```diff
@@ -0,0 +1,104 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.​integration.redis
+---
+
+# prometheus.integration.redis
+The `prometheus.integration.redis` component embeds
+[redis_exporter](https://github.com/oliver006/redis_exporter) for collecting metrics from a Redis database.
+
+## Usage
+
+```river
+prometheus.integration.redis "LABEL" {
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
```

Comment:
I have similar concerns here with the length of descriptions. If something needs to be longer than a single sentence, can the remainder of the information be moved outside of the table?

For an example, check out `otelcol.exporter.otlp`, specifically the queue block: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.exporter.otlp/#queue-block